### DISCUSSION TITLE
Jupyter Server

### DISCUSSION THREAD

**Discussion by vwolf89** [cooperrc/computational-mechanics#49] [Q&A]
Anyone else getting an 403 : Forbidden error every time you click Sign in with GitHub on JupyterHub and try to sign in? I've never been able to get my repository cloned because I can't get past this error. Tried clearing my cookies, my history, and using a different internet browser. FWIW I have Jupyter Notebook through anaconda3 logged in under my usual university email address not UCONN.

**Comment by cooperrc** [comment]
Make sure you're using UConn netid + UConn password, otherwise it will give you a `403` error (it means your username is not included in the use list). Otherwise, I'll doublecheck that your netid is in the userlist

**Comment by vwolf89** [comment]
I could only get that far once and I got a 400 error. The rest of the times I’ve tried I click the orange sign in rectangle and immediately get the 403 error.

Valerie Wolf
Senior Mechanical Engineering Student
University of Colorado - Denver, CEDC
***@***.***
720-350-0902
________________________________

**Comment by cooperrc** [comment]
Do you get the error 

## Here?:
![image](https://user-images.githubusercontent.com/11371428/173402587-14c09853-285f-44d9-ad46-eb5fbbdeb073.png)
## or Here?:
![image](https://user-images.githubusercontent.com/11371428/173402631-66fe4cf7-59d6-457c-91fc-9dfdc3df2697.png)

**Comment by vwolf89** [comment]
I’ve also allowed pop-ups and everything else in my website settings. It is saying there is an error with the certificate.

Valerie Wolf
Senior Mechanical Engineering Student
University of Colorado – Denver, CEDC
***@***.******@***.***>

From: Wolf, ***@***.***>
Sent: Monday, June 13, 2022 1:49 PM
To: ***@***.***>
Subject: RE: [cooperrc/computational-mechanics] Jupyter Server (Discussion #49)

The first time I tried to log in, I got the error 400 after the second picture. I was able to put my UCONN id in and then got the error message. Since that first time I get an error after the first picture. I’ve cleared my history and my cookies. I’ve also tried on chrome and Microsoft Edge.

Valerie Wolf
Mechanical Engineering Student
University of Colorado – Denver, CEDC
***@***.******@***.***>

**Comment by cooperrc** [comment]
The certificate error is something on my end that I don't know how to fix right now. 

There should be an option to choose "advanced" -> "Proceed to compmech.uconn.edu anyway" at the bottom of the page.

**Comment by vwolf89** [comment]
I have Jupyter Notebook on my laptop through Anaconda 3 that I have logged out of with my regular school account. This is the page I get after I log out and if I put my UCONN id into it, it just gives me this same page.

Valerie Wolf
Senior Mechanical Engineering Student
University of Colorado – Denver, CEDC
***@***.******@***.***>

From: Wolf, ***@***.***>
Sent: Monday, June 13, 2022 1:53 PM
To: ***@***.***>
Subject: RE: [cooperrc/computational-mechanics] Jupyter Server (Discussion #49)

I’ve also allowed pop-ups and everything else in my website settings. It is saying there is an error with the certificate.

Valerie Wolf
Senior Mechanical Engineering Student
University of Colorado – Denver, CEDC
***@***.******@***.***>

From: Wolf, ***@***.***>
Sent: Monday, June 13, 2022 1:49 PM
To: ***@***.***>
Subject: RE: [cooperrc/computational-mechanics] Jupyter Server (Discussion #49)

The first time I tried to log in, I got the error 400 after the second picture. I was able to put my UCONN id in and then got the error message. Since that first time I get an error after the first picture. I’ve cleared my history and my cookies. I’ve also tried on chrome and Microsoft Edge.

Valerie Wolf
Mechanical Engineering Student
University of Colorado – Denver, CEDC
***@***.******@***.***>

**Comment by vwolf89** [comment]
The first time I tried to log in, I got the error 400 after the second picture. I was able to put my UCONN id in and then got the error message. Since that first time I get an error after the first picture. I’ve cleared my history and my cookies. I’ve also tried on chrome and Microsoft Edge.

Valerie Wolf
Mechanical Engineering Student
University of Colorado – Denver, CEDC
***@***.******@***.***>
